### PR TITLE
update some formats in python API doc

### PR DESCRIPTION
**Pull request by helenlly**
update some tables format using  ">>>" with the table codes below:

"
 .. list-table::
   :widths: 50 50
   :header-rows: 1

   * - Name
     - Description
    .......
"

you can see the diff in the links below:

 current format:  https://analytics-zoo.readthedocs.io/en/latest/doc/PythonAPI/Orca/orca.html
 updated format:  https://testhelendoc.readthedocs.io/en/latest/doc/PythonAPI/Orca/orca.html

### CODE REVIEW

File: pyzoo/zoo/orca/learn/bigdl/estimator.py [hunk 1]
```diff
@@ -466,23 +466,42 @@ def get_validation_summary(self, tag=None):
...
-        >>> 'Top5Accuracy'              |   'Top5Accuracy'
-        >>> 'TreeNNAccuracy'            |   'TreeNNAccuracy()'
-        >>> 'MeanAveragePrecision'      |   'MAP@k' (k is Top-k) (BigDL)
-        >>> 'MeanAveragePrecision'      |   'PascalMeanAveragePrecision' (Zoo)
-        >>> 'StatelessMetric'           |   '${name}'
-
-        :param tag: The string variable represents the scalar wanted
+.. list-table::
+   :widths: 50 50
+   :header-rows: 1
```
Comment (hkvision) [inline pyzoo/zoo/orca/learn/bigdl/estimator.py:471]: are there any other options to do this?
It is strange to include such meta code here...

Comment (helenlly) [reply pyzoo/zoo/orca/learn/bigdl/estimator.py:471]: > are there any other options to do this?
> It is strange to include such meta code here...

 or still use the current format ...

Comment (helenlly) [reply pyzoo/zoo/orca/learn/bigdl/estimator.py:471]: or not using table, just  add blank lines  between , like this:


        'Accuracy'                  |   'Top1Accuracy'

        'BinaryAccuracy'            |   'Top1Accuracy'

        'CategoricalAccuracy'       |   'Top1Accuracy'

        'SparseCategoricalAccuracy' |   'Top1Accuracy

.......

**Comment by helenlly** [comment]
if remove table or >>>, it looks like :
https://testhelendoc.readthedocs.io/en/latest/doc/PythonAPI/Orca/orca.html#module-zoo.orca.learn.bigdl.estimator

pls check if can acceptable. Or we need think another way to describe the metric and tags. e.g.

 1. left:  ‘Accuracy’ 
 
     right: ‘Top1Accuracy’

 2. left: ‘BinaryAccuracy’
 
    right: ‘Top1Accuracy’
........

@hkvision @yangw1234

**Comment by jason-dai** [comment]
> if remove table or >>>, it looks like :
> https://testhelendoc.readthedocs.io/en/latest/doc/PythonAPI/Orca/orca.html#module-zoo.orca.learn.bigdl.estimator
> 
> pls check if can acceptable. Or we need think another way to describe the metric and tags. e.g.
> 
> 1. left:  ‘Accuracy’
>    right: ‘Top1Accuracy’
> 2. left: ‘BinaryAccuracy’
>    right: ‘Top1Accuracy’
>    ........
> 
> @hkvision @yangw1234

I actually like the original version. One question: why does `>>>` show up in the rendered page?

**Comment by jason-dai** [comment]
> > if remove table or >>>, it looks like :
> > https://testhelendoc.readthedocs.io/en/latest/doc/PythonAPI/Orca/orca.html#module-zoo.orca.learn.bigdl.estimator
> > pls check if can acceptable. Or we need think another way to describe the metric and tags. e.g.
> > 
> > 1. left:  ‘Accuracy’
> >    right: ‘Top1Accuracy’
> > 2. left: ‘BinaryAccuracy’
> >    right: ‘Top1Accuracy’
> >    ........
> > 
> > @hkvision @yangw1234
> 
> I actually like the original version. One question: why does `>>>` show up in the rendered page?

In addition, we should make the metrics names consistent with the TensorBoard names

**Comment by jason-dai** [comment]
> > if remove table or >>>, it looks like :
> > https://testhelendoc.readthedocs.io/en/latest/doc/PythonAPI/Orca/orca.html#module-zoo.orca.learn.bigdl.estimator
> > pls check if can acceptable. Or we need think another way to describe the metric and tags. e.g.
> > 
> > 1. left:  ‘Accuracy’
> >    right: ‘Top1Accuracy’
> > 2. left: ‘BinaryAccuracy’
> >    right: ‘Top1Accuracy’
> >    ........
> > 
> > @hkvision @yangw1234
> 
> I actually like the original version. One question: why does `>>>` show up in the rendered page?

In addition, we should make the metrics names consistent with the TensorBoard names @yangw1234

**Comment by helenlly** [comment]
> > if remove table or >>>, it looks like :
> > https://testhelendoc.readthedocs.io/en/latest/doc/PythonAPI/Orca/orca.html#module-zoo.orca.learn.bigdl.estimator
> > pls check if can acceptable. Or we need think another way to describe the metric and tags. e.g.
> > 
> > 1. left:  ‘Accuracy’
> >    right: ‘Top1Accuracy’
> > 2. left: ‘BinaryAccuracy’
> >    right: ‘Top1Accuracy’
> >    ........
> > 
> > @hkvision @yangw1234
> 
> I actually like the original version. One question: why does `>>>` show up in the rendered page?

 It is the prompt of the interactive Python interpreter.  the words followed" >>>" will be showed in the doc website as same as what they look in the py file.  So here,  ">>>" will help to keep the blank spaces existing in the row and make the words format look neat,  but '>>>' will be showed too.

**Comment by helenlly** [comment]
keep current display format and cancel the PR

**Pull request closed without merging**